stock_prices/stock_prices.py

```diff
@@ -1,38 +1,23 @@
 #!/usr/bin/python
 
 import argparse
+#Got a solution that works, but not optimal. O(n^2)
+def find_max_profit (stock_prices):
+  max = float("-inf")
+  for i in range(0, len(stock_prices)):
```

Comment:
yep, this works but it's an O(n**2) solution. after you're done with MVP, see if you can come up with an O(n) algorithm